src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/AncientAltarTask.java

```diff
@@ -129,8 +129,7 @@ private void checkPedestal(Block pedestal) {
 
             itemLock.remove(item);
             item.remove();
-
-            pedestal.removeMetadata("item_placed", SlimefunPlugin.instance);
```

Comment:
Why was this removed?

Reply:
I could not find anywhere that this was actually added

Reply:
Hmm true. Weird...
Since there hasn't been any bug reports or complaints about the altars recently I will just assume that this is safe to remove, some probably added this at some point and removed it half-heartedly, leaving these things behind. Guess it wasn't that important then.